client/src/app/pages/applications/applications-table-analyze/applications-table-analyze.tsx

```diff
@@ -454,6 +454,12 @@ export const ApplicationsTableAnalyze: React.FC = () => {
     tasks.some((task) => task.application?.id === app.id)
   );
 
+  const getTaskState = (application: Application) => {
+    const task = getTask(application);
+    if (task && task.state) return task.state;
+    return "No task";
+  };
+
```

Comment:
should be able to remove this now

```suggestion
```

Reply:
@gitdallas , I forgot to save the file ! Thanks